be/src/storage/tablet_updates.cpp

```diff
@@ -2944,9 +2944,10 @@ size_t TabletUpdates::_get_rowset_num_deletes(const Rowset& rowset) {
 }
 
 StatusOr<ExtraFileSize> TabletUpdates::_get_extra_file_size() const {
+    ExtraFileSize ef_size;
+#if !defined(ADDRESS_SANITIZER)
```

Comment:
could be related to `tablet_path` that is constructed from a temp c_str?

Reply:
No, I have to try this before, but it still fail with same reason.

Reply:
let asan crash immediately, check in the core, find what's there for the mentioned address

Reply:
Ok, I can't reproduce it now, let's wait for QA reproduce it again.